common/src/paused_program_storage.rs

```diff
@@ -248,7 +248,7 @@ pub trait PausedProgramStorage: super::ProgramStorage {
                 };
 
                 // it means that the program has been already resumed within another session
```

Comment:
What about the same check during push?

Reply:
Yep, good suggestion. It looks better to add it in the following PR

Reply:
> Yep, good suggestion. It looks better to add it in the following PR

Open an issue about that?